invalid conversion from 'char' to 'const char*'

````
/builddir/build/BUILD/percolator-rel-3-01/src/converters/MSToolkit/RAMP/ramp.cpp: In function 'RAMPREAL* readPeaks(RAMPFILE*, ramp_fileoffset_t)':
/builddir/build/BUILD/percolator-rel-3-01/src/converters/MSToolkit/RAMP/ramp.cpp:1601:29: error: invalid conversion from 'char' to 'const char*' [-fpermissive]
             pEndAttrValue = '\0';
                             ^~~~
/builddir/build/BUILD/percolator-rel-3-01/src/converters/MSToolkit/RAMP/ramp.cpp:1615:29: error: invalid conversion from 'char' to 'const char*' [-fpermissive]
             pEndAttrValue = '\0';
                             ^~~~
````

Presumably this is supposed to be truncating the strings, i.e. `*pEndAttrValue = '\0';` but instead it sets the pointer to null or fails to compile, depending which compiler you use.

--- a/src/converters/MSToolkit/RAMP/ramp.cpp
+++ b/src/converters/MSToolkit/RAMP/ramp.cpp
@@ -1598,8 +1598,8 @@ RAMPREAL* readPeaks(RAMPFILE* pFI, ramp_fileoffset_t lScanIndex) {
             const char* pEndAttrValue;
             pEndAttrValue = strchr(pBeginData + strlen("contentType=\"")
                 + 1, '\"');
-            pEndAttrValue = '\0';
-            fprintf(stderr, "%s Unsupported content type\n", pBeginData);
+            int len = pEndAttrValue - pBeginData;
+            fprintf(stderr, "%.*s Unsupported content type\n", len, pBeginData);
             return NULL;
           }
         }
@@ -1612,10 +1612,10 @@ RAMPREAL* readPeaks(RAMPFILE* pFI, ramp_fileoffset_t lScanIndex) {
             const char* pEndAttrValue;
             pEndAttrValue = strchr(pBeginData
                 + strlen("compressionType=\"") + 1, '\"');
-            pEndAttrValue = '\0';
+            int len = pEndAttrValue - pBeginData;
             fprintf(stderr,
-                    "%s Unsupported compression type\n",
-                    pBeginData);
+                    "%.*s Unsupported compression type\n",
+                    len, pBeginData);
             return NULL;
           }
         }

